src/libAtomVM/mailbox.c

```diff
@@ -292,15 +296,19 @@ bool mailbox_peek(Context *c, term *out)
 
     TRACE("Pid %i is peeking 0x%lx.\n", c->process_id, m->message);
```

Comment:
I think this no longer compiles as `m` has no `message` field.

Reply:
Fixed.